testdata/project-v3-with-deploy-image/controllers/busybox_controller_test.go

```diff
@@ -85,9 +85,7 @@ var _ = Describe("Busybox controller", func() {
 				}
 
 				err = k8sClient.Create(ctx, busybox)
-				if err != nil {
-					Expect(err).To(Not(HaveOccurred()))
-				}
+				Expect(err).To(Not(HaveOccurred()))
```

Comment:
Hi @LittleWat, 

Thank you for the contribution 🥇.

But this change cannot be done in the samples. the samples are generated automatically when we run `make generate`, see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/testdata/generate.sh

Therefore, this change needs to be done in the plugin implementation: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller-test.go#L138-L140  

Could you please do that in the code implementation and run make generate to ensure that all samples will be updated according to, squash the commits and push it so we are able to get this one merged?

Reply:
Also, please note that you need assign to the CLA. https://github.com/kubernetes-sigs/kubebuilder/pull/2838#issuecomment-1197606995

Reply:
@camilamacedo86 Thank you for quick and detailed review! Oh, these test codes are generated, cool!

I edited the original template file and  did `make generate` and committed!
I also signed!